### PR TITLE
user profile

### PR DESCRIPTION
**Pull request by mariehposa**
#### What does this PR do?
user profile page
#### Description of Task to be completed?
Test

#### How should this be manually tested?
yarn start or npm start
#### Any background context you want to provide?

- [*] I have performed a self-review of my own code
- [*] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [*] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [*] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/templates/UserProfileForm.jsx [hunk 1]
```diff
@@ -160,7 +160,7 @@ const UserProfileForm = ({initialState}) => {
                           Cancel
                         </Button>
                       </Link>
-                      <Button color="green" type="submit">
+                      <Button color="green" to="/dashboard/profile" type="submit">
```
Comment (attaradev) [inline src/components/templates/UserProfileForm.jsx:163]: Why add the `to` on the submit button @mariehposa

File: src/App.jsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ import ParticipantSubmissionPage from "./components/views/ParticipantSubmissionP
 import "react-toastify/dist/ReactToastify.css";
 import PageNotFound from "./components/views/PageNotFound";
 import UserProfileFormPage from "./components/views/UserProfileFormPage";
+import UserProfilePage from "./components/views/userProfilePage";
```
Comment (PascalUlor) [inline src/App.jsx]: Always start the naming of your component files in caps and in camel case
"./components/views/userProfilePage"; 
should be
"./components/views/UserProfilePage";
This error is disrupting our Netlify build

Comment (PascalUlor) [reply src/App.jsx]: branch naming does not follow convention. Please prepend
`feat` or `feature` if it's a feature branch
`chore` if it's a chore branch
`bug` or `bugfix` if it's a bugfix branch

**Review by PascalUlor — COMMENTED** [review]
Nice job on the profile page design and logic Mariam.
Please attend to my feedback before I can merge.

File: src/App.jsx (hunk 2 above)
Comment (PascalUlor) [reply src/App.jsx]: We need to work on rendering and incrementing the page count on the frontend

File: src/components/templates/UserProfileForm.jsx (hunk 1 above)
Comment (mariehposa) [reply src/components/templates/UserProfileForm.jsx]: That was a mistake and could be found on other submit buttons too.

Comment (mariehposa) [reply src/components/templates/UserProfileForm.jsx:163]: I didn't work on the user profile form, only the user profile.

File: src/App.jsx (hunk 2 above)
Comment (mariehposa) [reply src/App.jsx]: It was probably a mistake, will work on it.

**Pull request closed without merging**